test/javascripts/acceptance/keyboard-shortcuts-test.js.es6

```diff
@@ -0,0 +1,85 @@
+import { acceptance } from "helpers/qunit-helpers";
+import DiscourseURL from "discourse/lib/url";
+
+acceptance("Keyboard Shortcuts", { loggedIn: true });
+
+test("go to first suggested topic", async assert => {
+  sandbox.stub(DiscourseURL, "routeTo");
+
+  /*
+   * No suggested topics exist.
+   */
+
+  server.get("/t/9/last.json", () => [
+    200,
+    { "Content-Type": "application/json" },
+    {}
+  ]);
+
+  await visit("/t/this-is-a-test-topic/9");
+  await keyEvent(document, "keypress", "g".charCodeAt(0));
+  await keyEvent(document, "keypress", "s".charCodeAt(0));
+  assert.ok(DiscourseURL.routeTo.getCalls().length == 0);
+
+  /*
+   * Suggested topics elements exist.
+   */
+
+  await visit("/t/internationalization-localization/280");
+  await keyEvent(document, "keypress", "g".charCodeAt(0));
+  await keyEvent(document, "keypress", "s".charCodeAt(0));
+  assert.ok(
```

Comment:
Since this is an acceptance test, we should instead test that the URL has changed to new topic's URL.